pkg/v16/key/key_test.go

```diff
@@ -221,6 +221,34 @@ func Test_DNSIP(t *testing.T) {
 	}
 }
 
+func Test_ClusterConfigMapName(t *testing.T) {
+	testCases := []struct {
+		description    string
+		guestConfig    v1alpha1.ClusterGuestConfig
```

Comment:
```suggestion
		clusterConfig    v1alpha1.ClusterGuestConfig
```

The CRD name is ugly and uses the old guest cluster naming. Lets try and keep `guest` to a minimum.